control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -4225,6 +4225,92 @@ paths:
                 $ref: "#/components/schemas/RestJsonError"
       security:
         - JWT: []
+  /states:
+    get:
+      tags:
+        - States
+      operationId: get_states
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Specs"
```

Comment:
You've missed this.

Reply:
Well spotted - thanks. Changed it to states.